scripts/utils/formatCode.js

```diff
@@ -1,11 +1,22 @@
 "use strict";
-const prettier = require("prettier");
 
-module.exports = function formatCode(code, filename) {
-  filename = filename || __filename;
-  const prettierConfig = prettier.resolveConfig.sync(filename);
-  prettierConfig.filepath = filename;
-  prettierConfig.parser = "babel";
+// TODO: Remove this `if` in Babel 8
+// Prettier only supports Node.js 10+, so we can fallback to not formatting
+// o CI on older Node.js versions
 
-  return prettier.format(code, prettierConfig);
-};
+if (process.env.CI && parseInt(process.versions.node, 10) < 10) {
```

Comment:
We use prettier format the generated type definition files. However, we don't really need to do this on CI and we can skip it on Node.js 6 and 8 without introducing problems.

Reply:
looks good for me, was thinking about disabling code generation completely - but, I think, having it without prettier is a better option